src/app/components/normal-component/pick-game/pick-game.component.ts

```diff
@@ -35,7 +35,7 @@ class GameDescription {
 
     public static readonly P4: string = $localize`The classical 4 in a row game!`;
 
-    public static readonly PENTAGO: string = $localize`Drop a piece, then rotate the board, first to align 5 of his pieces win!`;
+    public static readonly PENTAGO: string = $localize`Drop a piece, then rotate the board, be the first to align 5 of your pieces to win!`;
 
```

Comment:
"Drop a piece then rotate the board. The first to align 5 of his pieces win!"
C'est mieux point de vue rythme ? C'est sensé être un peu un slogan accrocheur.

Reply:
Oui, mais pas "his" car le joueur n'est pas spécialement un homme :D

Reply:
Je dirais que comme c'est pas un tuto mais un slogan, mieux vaut dire "first player to align 5 pieces win", ça pourrais être implicite, les gens connaissent bien les N-in-a-line ?

Reply:
Je suis pas convaincu que « Soyez le premier à aligner 5 pièces pour gagner ! » sonne comme un « tuto » alors que « Le premier à aligner 5 pièces gagne ! » sonne comme un slogan, pour moi c'est kif kif entre les deux